src/ground_truth_nsrts.py

```diff
@@ -862,6 +864,375 @@ def placeontable_sampler(state: State, goal: Set[GroundAtom],
     return nsrts
 
 
+def _get_repeated_nextto_painting_gt_nsrts() -> Set[NSRT]:
```

Comment:
most of this should be the same as the painting gt nsrts right? can you refactor to remove the duplicate code? I'll take another look at this part after that